### PR TITLE
msm8937-common: update p files

### PR DESCRIPTION
**Pull request by JarlPenguin**

### CODE REVIEW

**Review by JarlPenguin — COMMENTED** [review]
+1

**Review by JarlPenguin — COMMENTED** [review]
+1

File: proprietary-files.txt [hunk 1]
```diff
@@ -535,191 +747,22 @@ vendor/lib/mmi_sysinfo.so
...
-vendor/lib64/libqmi_client_qmux.so
-vendor/lib64/libqmi_common_so.so
-vendor/lib64/libqmi_csi.so
-vendor/lib64/libqmi_csvt_srvc.so
-vendor/lib64/libqmi_encdec.so
-vendor/lib64/libqmimotext.so
-vendor/lib64/libqmiservices.so
 vendor/lib64/libqti-at.so
 vendor/lib64/libqti-gt-prop.so
 vendor/lib64/libqti-gt.so
```
Comment (wiktorek140) [inline proprietary-files.txt:725]: In theory its qti perf files

File: proprietary-files.txt [hunk 2]
```diff
@@ -535,191 +747,22 @@ vendor/lib/mmi_sysinfo.so
...
-vendor/lib64/libllvm-glnext.so
-vendor/lib64/libllvm-qcom.so
-vendor/lib64/libllvm-qgl.so
-vendor/lib64/liblqe.so
-vendor/lib64/libmdmcutback.so
-vendor/lib64/libmdmdetect.so
-vendor/lib64/libmm-color-convertor.so
-vendor/lib64/libmm-disp-apis.so
-vendor/lib64/libmm-hdcpmgr.so
 vendor/lib64/libmm-qdcm.so
```
Comment (wiktorek140) [inline proprietary-files.txt:722]: Display section

File: proprietary-files.txt [hunk 3]
```diff
@@ -535,191 +747,22 @@ vendor/lib/mmi_sysinfo.so
...
-vendor/lib64/libadreno_utils.so
-vendor/lib64/libadropbox.so
-vendor/lib64/libaudioalsa.so
-vendor/lib64/libaudioparsers.so
-vendor/lib64/libbccQTI.so
-vendor/lib64/libbtnv.so
 vendor/lib64/libc2d30-a3xx.so
 vendor/lib64/libc2d30-a4xx.so
 vendor/lib64/libc2d30-a5xx.so
 vendor/lib64/libc2d30_bltlib.so
```
Comment (wiktorek140) [inline proprietary-files.txt:718]: Adreno blobs

File: proprietary-files.txt [hunk 4]
```diff
@@ -435,85 +703,29 @@ vendor/lib/libmmiipstreammmihttp.so
...
-vendor/lib/libwfdcommonutils.so
-vendor/lib/libwfdhdcpcp.so
-vendor/lib/libwfdmmsink.so
-vendor/lib/libwfduibcsink.so
-vendor/lib/libwfduibcsinkinterface.so
-vendor/lib/libwms.so
-vendor/lib/libwqe.so
-vendor/lib/libxml.so
-vendor/lib/mediadrm/libwvdrmengine.so
 vendor/lib/mmi_audio.so
```
Comment (wiktorek140) [inline proprietary-files.txt]: Are all mmi_ files are used?

File: proprietary-files.txt [hunk 5]
```diff
@@ -435,85 +703,29 @@ vendor/lib/libmmiipstreammmihttp.so
...
 vendor/lib/libsmemlog.so
-vendor/lib/libssd.so
 vendor/lib/libsubsystem_control.so
-vendor/lib/libsystem_health_mon.so
-vendor/lib/libthermalclient.so
-vendor/lib/libthermalioctl.so
-vendor/lib/libtinyxml2_1.so
 vendor/lib/libuy.so
 vendor/lib/libvendorconn.so
 vendor/lib/libvoice-svc.so
```
Comment (wiktorek140) [inline proprietary-files.txt:709]: audio probably

File: proprietary-files.txt [hunk 6]
```diff
@@ -435,85 +703,29 @@ vendor/lib/libmmiipstreammmihttp.so
...
-vendor/lib/libqti-perfd-client.so
-vendor/lib/libqti_performance.so
-vendor/lib/librcc.so
 vendor/lib/libremosaic_daemon.so
-vendor/lib/librpmb.so
-vendor/lib/librs_adreno.so
-vendor/lib/librs_adreno_sha1.so
 vendor/lib/libscalar.so
 vendor/lib/libscale.so
 vendor/lib/libsd_sdk_display.so
```
Comment (wiktorek140) [inline proprietary-files.txt]: display probably

File: proprietary-files.txt [hunk 7]
```diff
@@ -435,85 +703,29 @@ vendor/lib/libmmiipstreammmihttp.so
...
-vendor/lib/libqmi_cci.so
-vendor/lib/libqmi_client_helper.so
-vendor/lib/libqmi_client_qmux.so
-vendor/lib/libqmi_common_so.so
-vendor/lib/libqmi_csi.so
-vendor/lib/libqmi_csvt_srvc.so
-vendor/lib/libqmi_encdec.so
-vendor/lib/libqmiservices.so
 vendor/lib/libqti-at.so
 vendor/lib/libqti-gt.so
```
Comment (wiktorek140) [inline proprietary-files.txt:699]: i assume perf files

File: proprietary-files.txt [hunk 8]
```diff
@@ -435,85 +703,29 @@ vendor/lib/libmmiipstreammmihttp.so
...
-vendor/lib/libmmrtpdecoder.so
-vendor/lib/libmmrtpencoder.so
 vendor/lib/libmmsw_detail_enhancement.so
 vendor/lib/libmmsw_math.so
 vendor/lib/libmmsw_opencl.so
 vendor/lib/libmmsw_platform.so
 vendor/lib/libmodalityservice_jni.so
-vendor/lib/libnetmgr.so
-vendor/lib/liboemcrypto.so
 vendor/lib/libomx-dts.so
```
Comment (wiktorek140) [inline proprietary-files.txt:694]: omx / audio

File: proprietary-files.txt [hunk 9]
```diff
@@ -182,249 +644,55 @@ etc/firmware/rampatch_tlv_3.0.tlv
...
 vendor/lib/libasn1crtx.so
-vendor/lib/libaudioalsa.so
-vendor/lib/libaudioparsers.so
-vendor/lib/libbccQTI.so
-vendor/lib/libbson.so
-vendor/lib/libbtnv.so
 vendor/lib/libc2d30-a3xx.so
 vendor/lib/libc2d30-a4xx.so
 vendor/lib/libc2d30-a5xx.so
 vendor/lib/libc2d30_bltlib.so
```
Comment (wiktorek140) [inline proprietary-files.txt:674]: Adreno blobs.

File: proprietary-files.txt [hunk 10]
```diff
@@ -182,249 +644,55 @@ etc/firmware/rampatch_tlv_3.0.tlv
...
-vendor/lib/libOpenCL.so
-vendor/lib/libQSEEComAPI.so
-vendor/lib/libQtiTether.so
-vendor/lib/libRSDriver_adreno.so
+vendor/lib/libCommandSvc.so
+vendor/lib/libExtendedExtractor.so
+vendor/lib/libFIDOKeyProvisioning.so
+vendor/lib/libFidoCrypto.so
+vendor/lib/libFidoCryptoJNI.so
+vendor/lib/libFidoSuiJNI.so
```
Comment (wiktorek140) [inline proprietary-files.txt:661]: Encrypting probably

File: proprietary-files.txt [hunk 11]
```diff
@@ -182,249 +644,55 @@ etc/firmware/rampatch_tlv_3.0.tlv
...
-vendor/bin/qseecomd
 vendor/bin/qseeproxydaemon
 vendor/bin/qseeproxysampledaemon
-vendor/bin/qti
 vendor/bin/radish
-vendor/bin/rmt_storage
 vendor/bin/ssr_setup
-vendor/bin/tftp_server
 vendor/bin/xtwifi-client
 vendor/bin/xtwifi-inet-agent
```
Comment (wiktorek140) [inline proprietary-files.txt]: gps, location based on wifi

File: proprietary-files.txt (hunk 4 above)
Comment (JarlPenguin) [reply proprietary-files.txt]: I don't know about that, last time I looked in potter vendor they were updated from stock Nougat

File: proprietary-files.txt [hunk 12]
```diff
@@ -535,191 +747,22 @@ vendor/lib/mmi_sysinfo.so
...
-vendor/lib64/libsdm-color.so
-vendor/lib64/libsdm-diag.so
-vendor/lib64/libsdm-disp-vndapis.so
-vendor/lib64/libsdmextension.so
-vendor/lib64/libsensorndkbridge.so
-vendor/lib64/libsettings.so
 vendor/lib64/libslimclient.so
 vendor/lib64/libsmemlog.so
-vendor/lib64/libssd.so
 vendor/lib64/libsubsystem_control.so
```
Comment (wiktorek140) [inline proprietary-files.txt:729]: Its look like audio lib but not sure

File: proprietary-files.txt [hunk 13]
```diff
@@ -535,191 +747,22 @@ vendor/lib/mmi_sysinfo.so
...
 vendor/lib64/libc2d30_bltlib.so
-vendor/lib64/libcne.so
-vendor/lib64/libcneapiclient.so
-vendor/lib64/libcneoplookup.so
-vendor/lib64/libcneqmiutils.so
-vendor/lib64/libconfigdb.so
 vendor/lib64/libdataitems.so
-vendor/lib64/libdiag.so
-vendor/lib64/libdiag_system.so
 vendor/lib64/libdiagjni.so
```
Comment (wiktorek140) [inline proprietary-files.txt:720]: With other diag libs

**Comment by JarlPenguin** [comment]
maybe we should try cleaning up vendor?

**Comment by wiktorek140** [comment]
Not that bad idea

**Pull request closed without merging**